org.mfek.glif.yml

```diff
@@ -0,0 +1,44 @@
+app-id: org.mfek.glif
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
```

Comment:
please, don't use rust-nightly

Reply:
Due to the features used by the program, rust-nightly is required. If it's possible to do so, we could freeze the rust-nightly version to today's (I don't know how), but I'm unsure of the amount of work required to use the stable branch instead... Paging @ctrlcctrlv

Reply:
@hfiguiere Is this a Flathub policy, and what's the reason for it?

_cf._ [MFEK/glif@`53f4523`:/src/main.rs <sup>¶</sup> line № 6](https://github.com/MFEK/glif/blob/53f452363e97d402a5dbc9a02ce0d3177909945e/src/main.rs#L6) _et seq._:

```rust
#![feature(
    type_alias_impl_trait,
    local_key_cell_methods,
    trait_alias,
    const_trait_impl,
    generic_const_exprs,
    adt_const_params
)]
```

These are all in extensive use, especially `generic_const_exprs` (rust-lang/rust#76560) but also `adt_const_params` (rust-lang/rust#95174), neither of which are on track for _imminent_ stabilization (especially not the former, open since 2020).

If reproducibility is the concern as I suspect, can't @Subject38 just pick a date and use that nightly?

Reply:
org.freedesktop.Sdk.Extension.rust-nightly.YYYY.MM.DD ?

Reply:
NO

Reply:
By bundle, do you mean we should be downloading that specific version using rustup in the build script?

Reply:
yes

Reply:
> NO

![o_12](https://github.com/flathub/flathub/assets/838783/8dab2c85-302c-4e53-9526-5d6bcc99b387)